main.go

```diff
@@ -19,7 +19,6 @@ import (
 	"github.com/livepeer/catalyst-api/cluster"
 	"github.com/livepeer/catalyst-api/config"
 	mistapiconnector "github.com/livepeer/catalyst-api/mapic"
-	"github.com/livepeer/catalyst-api/middleware"
```

Comment:
should be just commented out (so revert commit would be just uncommenting, and not searching for what 'middleware' package was

Reply:
Yep good catch, fixed.